recipes/smart_open/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "smart_open" %}
+{% set version = "1.3.5" %}
+{% set sha256 = "379bec2c08c518a30fe58e781abcae8d4e58bc0024c950f13d55ebcc899e3aba" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - boto >=2.32
+    - bz2file
+    - requests
```

Comment:
Please drop any of these that are not strictly required for the build. Most likely only `python` and `setuptools`.